web/modules/custom/rdf_entity/src/Entity/RdfEntitySparqlStorage.php

```diff
@@ -656,34 +659,64 @@ protected function doSave($id, EntityInterface $entity) {
     $target_graph = $this->getGraphHandler()->getTargetGraphFromEntity($entity);
     $graph_uri = $this->getBundleGraphUri($bundle, $target_graph);
 
-    $insert = '';
+    $graph = new Graph($graph_uri);
```

Comment:
I would prefer to have this as a static method, part of rdf Query class or the RdfSparqlStorage class that will return a new graph and maybe we can handle exceptions there in the future.